WordPress/Classes/ViewRelated/Post/Prepublishing/PrepublishingSocialAccountsTableFooterView.swift

```diff
@@ -0,0 +1,108 @@
+                                                               showsWarning: showsWarning,
+                                                               onButtonTap: onButtonTap)
+        let viewToEmbed = UIView.embedSwiftUIView(footerView)
+        contentView.addSubview(viewToEmbed)
+        contentView.pinSubviewToAllEdges(viewToEmbed)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+// MARK: - SwiftUI View
+
+struct PrepublishingSocialAccountsFooterView: View {
+
+    @State var remaining: Int
+    @State var showsWarning: Bool
+    var onButtonTap: (() -> Void)?
+
+    var body: some View {
+        VStack(spacing: 16.0) {
+            remainingSharesText
+            subscribeButton
+        }
+        .padding(EdgeInsets(top: 24.0, leading: 0, bottom: 0, trailing: 0))
+    }
+
+    var remainingSharesText: some View {
+        HStack(alignment: .top, spacing: 6.0) {
+            if showsWarning {
+                Image("icon-warning")
+                    .resizable()
+                    .frame(width: 16.0, height: 16.0)
+                    .padding(2.0)
+                    .accessibilityElement()
+                    .accessibilityLabel(Constants.warningIconAccessibilityText)
+            }
+            Text(String(format: Constants.remainingSharesLabelTextFormat, remaining))
+                .font(.callout)
+                .foregroundColor(Color(showsWarning ? Constants.warningColor : .label))
+        }
+        .accessibilityElement(children: .combine)
+        .accessibilityAddTraits(.isStaticText)
+    }
+
+    var subscribeButton: some View {
+        Button {
+            onButtonTap?()
+        } label: {
+            Text(Constants.subscribeButtonText)
+                .padding(12.0)
+                .frame(maxWidth: .infinity) // needs to be set here to make the button stretch full-width.
+        }
+        .buttonStyle(SubscribeButtonStyle())
+    }
+
+    private struct SubscribeButtonStyle: ButtonStyle {
+        func makeBody(configuration: Configuration) -> some View {
+            configuration.label
+                .font(Constants.buttonLabelFont)
+                .foregroundColor(.white)
+                .background(Color(configuration.isPressed ? Constants.buttonHighlightedColor : Constants.buttonColor))
+                .clipShape(RoundedRectangle(cornerRadius: 8.0))
+        }
+    }
+
+    private enum Constants {
+        static let buttonLabelFont = Font.title3.weight(.semibold)
+        static let buttonColor = UIColor.primary
+        static let buttonHighlightedColor = UIColor.muriel(color: .jetpackGreen, .shade70)
+        static let warningColor = UIColor.muriel(color: MurielColor(name: .yellow, shade: .shade50))
+
+        static let remainingSharesLabelTextFormat = NSLocalizedString(
+            "prepublishing.socialAccounts.footer.remainingShares.text",
+            value: "%1$d social shares remaining in the next 30 days",
```

Comment:
Not an issue for this PR but I wonder if we should create a plural and non-plural version of this string now that we've removed `/<ShareLimit>` from it.

Reply:
Good point. I'll consider this in the next PR!